app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,69 @@
+class IngredientsController < ApplicationController
+  # GET /ingredients/new
+  def new
+    @foods = current_user.foods.all
+    @recipe = current_recipe
+    @ingredient = Ingredient.new
+  end
```

Comment:
You changed the name of the controller and model to Ingredient which is currently affecting my previous work, I had created a controller and model to continue your work yesterday.
kindly look into this.
![image](https://user-images.githubusercontent.com/43564324/157683929-7abeb478-06cd-469b-a55b-6848a21d3da5.png)